nano/secure/common.hpp

```diff
@@ -360,14 +360,14 @@ class election_status final
 {
 public:
 	std::shared_ptr<nano::block> winner;
-	nano::amount tally;
-	nano::amount final_tally;
-	std::chrono::milliseconds election_end;
-	std::chrono::milliseconds election_duration;
-	unsigned confirmation_request_count;
-	unsigned block_count;
-	unsigned voter_count;
-	election_status_type type;
+	nano::amount tally{ 0 };
+	nano::amount final_tally{ 0 };
+	std::chrono::milliseconds election_end{ std::chrono::duration_cast<std::chrono::milliseconds> (std::chrono::system_clock::now ().time_since_epoch ()) };
+	std::chrono::milliseconds election_duration{ std::chrono::duration_values<std::chrono::milliseconds>::zero () };
```

Comment:
Can't this be just `{ 0 }` or `{}`?

Reply:
I didn't check, I can't quite remember the rules for default initialization versus being uninitialized, this just seems more explicit.